a tiny addition

Ernestopheles opened the pull request:
... to make things clear

Ernestopheles committed:
a tiny addition

... to make things clear

fritzmg reviewed (changes requested):
This is wrong, the `make` commands are supposed to be executed from the root folder.

Ernestopheles commented:
If I send the command `make build-manual` in the root folder `contao`, I get this error:

> make: *** Keine Regel, um „build-manual“ zu erstellen.  Schluss.

fritzmg commented:
> If I send the command `make build-manual` in the root folder `contao`, I get this error:

What do you mean by root folder `contao`? You need to execute this in the repository's folder.

Ernestopheles committed:
clone into activel directory

My change will let clone no more into contao/docs but into contao. Then the make commands will work as desribed from the contao directory.

Ernestopheles committed:
Update _index.de.md

fritzmg reviewed (commented):
@Ernestopheles can you also provide the changes for the English version?

Ernestopheles committed:
Korrektur und Ergänzung

Ernestopheles commented:
Correction done: as root is contao/, no change of directory should be done, as we are already and still in the root directory.
enhancement: Described the way how to add changes to an previously opened PR

Ernestopheles committed:
Verschiebungen korrigiert

Ernestopheles committed:
Verschiebungen korrigiert

Ernestopheles committed:
Verschiebungen korrigiert

fritzmg commented on docs/manual/contributing/_index.de.md:
````diff
@@ -118,13 +118,13 @@ cd contao
 
 ```bash
-git clone --recurse-submodules https://github.com/DEIN-GITHUB-BENUTZERNAME/docs.git
+git clone --recurse-submodules https://github.com/DEIN-GITHUB-BENUTZERNAME/docs.git ./
````
This will fail if the current directory is not empty.

fritzmg commented on docs/manual/contributing/_index.de.md:
````diff
@@ -118,13 +118,13 @@ cd contao
 
 ```bash
-git clone --recurse-submodules git@github.com:DEIN-GITHUB-BENUTZERNAME/docs.git
+git clone --recurse-submodules git@github.com:DEIN-GITHUB-BENUTZERNAME/docs.git ./
````
This will fail if the current directory is not empty.

Ernestopheles replied:
The Manual asks to create a new directory contao...

fritzmg replied:
Ah I see what you mean now, because previously we already generated a new folder.

Ernestopheles replied:
Thx, @fritzmg for review and suggestions

fritzmg commented:
@Ernestopheles why do you want to change any of this?

fritzmg commented on docs/manual/contributing/_index.de.md:
````diff
@@ -298,3 +292,33 @@ Bestätige das Erstellen eines Pull requests mit »Create pull request«.
+git checkout patch-4
+```
+
+Ändere auf der betreffenden Datei (im `docs` Verzeichnis).
````
```suggestion
Bearbeite die Dateien, die geändert werden sollen.
```

Ernestopheles committed:
Zusammenfügung zweier PR

Ernestopheles committed:
Update docs/manual/contributing/_index.de.md

Co-authored-by: Fritz Michael Gschwantner <fmg@inspiredminds.at>

netzarbeiter reviewed (approved):
Thank you, Ernst
